examples/run_benchmarker.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2021 MosaicML. All Rights Reserved.
+
+"""Entrypoint that runs the `Benchmarker` on a provided YAML hparams file."""
+import argparse
+import logging
+
+import composer
+from composer.benchmarker.benchmarker import Benchmarker
+from composer.benchmarker.benchmarker_hparams import BenchmarkerHparams
+from composer.trainer.trainer import Trainer
+from composer.trainer.trainer_hparams import TrainerHparams
+
+logger = logging.getLogger(__name__)
+
+
+def main() -> None:
+    logging.basicConfig()
+    logging.captureWarnings(True)
+
+    parser = argparse.ArgumentParser(parents=[BenchmarkerHparams.get_argparse(cli_args=True)])
```

Comment:
I don't think you should need to import argparse / parse here? yahp takes care of this automatically.